lib/mjsonwp.js

```diff
@@ -69,23 +69,43 @@ function checkParams (paramSets, jsonObj) {
   // some clients pass in an element id in the params
   optionalParams.push('id');
 
+  // if we have no required parameters, all is well
```

Comment:
This block of code could be moved up a couple lines to line 65, couldn't it? Because we don't care about setting up optional params if there are no required params.